x-pack/plugins/security/server/session_management/session_index.ts

```diff
@@ -241,19 +258,20 @@ export class SessionIndex {
    * @param sessionValue Session index value.
    */
   async update(sessionValue: Readonly<SessionIndexValue>) {
-    const { sid, metadata, ...sessionValueToStore } = sessionValue;
     try {
-      const { body: response, statusCode } = await this.options.elasticsearchClient.index(
-        {
-          id: sid,
-          index: this.indexName,
-          body: sessionValueToStore,
-          if_seq_no: metadata.sequenceNumber,
-          if_primary_term: metadata.primaryTerm,
-          refresh: 'wait_for',
-        },
-        { ignore: [409], meta: true }
-      );
+      let { body: response, statusCode } = await this.updateExistingSessionDocument(sessionValue, {
+        ignore404: true,
+      });
+
+      if (statusCode === 404) {
```

Comment:
__question:__ do we handle `404` for updates just to account for a case when the alias is missing, but the index exists? I mean if index doesn't exist then the session we're trying to update doesn't exist either...

Reply:
Yes my intention here was to handle the case where we have an index without an alias